var/spack/repos/builtin/packages/ecflow/package.py

```diff
@@ -60,19 +60,27 @@ class Ecflow(CMakePackage):
 
     depends_on("openssl@1:", when="@5:")
     depends_on("qt@5:", when="+ui")
-    depends_on("cmake@2.12.11:", type="build")
+    # Requirement to use the Python3_EXECUTABLE variable
```

Comment:
Looks like the comment is  out of place?

Reply:
No, cmake@3.16 provides -DPython3_EXECUTABLE